_sass/landing-page.scss

```diff
@@ -38,11 +22,73 @@
 .hero {
   color: #000;
   text-align: center;
-  padding: 50px;
+  padding: 10px;
+  padding-top: 100px;
 
   p {
     color: #000000;
   }
+
+  .btn-main {
+    background: $brand-color;
+    padding: 10px;
+    width: 200px;
+    font-size: 20px;
+    margin: auto;
+
+    @media #{$tablet} {
+      margin: initial;
+    }
+  }
+
+  .hero-image {
+    width: 100%;
+    margin-top: 20px;
+  }
+
+  .main-text {
+    text-align: center;
```

Comment:
Opinionated comment: Left align > centered text

<img width="951" alt="image" src="https://user-images.githubusercontent.com/14346926/66919212-36790a80-f021-11e9-9a4d-2f167e570e0c.png">

Reply:
I also like the left aligned one in terms of looks. But my argument against it would be the far away button, for a right-hander (in most cases) they would need to reach far out. 

That is why I always prefer keeping it in the center. Independent of hands, it is accessible. WDYT?